views/index.html

```diff
@@ -182,10 +182,11 @@ <h1>API Reference</h1>
             <a href="http://david-hu.com/2012/03/05/announcing-numbers-api.html"
               >David</a
             >
-            and <a href="http://mduan.com">Mack</a>. We want your feedback! Tell
-            us what you use this for.
+            and <a href="http://mduan.com">Mack</a>. We want your feedback!
+            Visit our
+            <a href="https://github.com/rithmschool/numbers_api">Github</a> to
```

Comment:
can you make it clear here that David and Mack are the creators, but are no longer active on the project? Something like "Created by David and Mack. Maintained by Rithm School. We want your feedback!" etc